Update README.md (#83)
epiforecasts/NCoVUtils: README.md

## Installation

Install the package and all dependencies with:

```r
remotes::install_github("epiforecasts/NCoVUtils", dependencies = TRUE)
```

## Usage

### Worldwide data

> There are three sources of worldwide, country-level data on cases and deaths.

Reviewer: I see only two? There is three functions but two sources?

  + ```NCoVUtils::get_who_cases(country = NULL, daily = TRUE))```
3. Extract daily international case and death counts compiled by ECDC using:
  + ```NCoVUtils::get_ecdc_cases()```

A further function for worldwide data extracts non-pharmaceutical interventions by country:

* ```NCoVUtils::get_interventions_data()```

And anonymised international patient linelist data can be imported and cleaned with:

* ```NCoVUtils::get_linelist()```

### Sub-national data

We have several functions to extract sub-national level data by country. These are typically at the admin-1 level, the largest regions available. We are also working on joining the data to standard georeferencing codes to allow easy mapping.

Currently we include functions for sub-national data in the following countries:

Europe

  +	Belgium

  +	France

  + Germany

  +	Italy

  +	Spain

  + United Kingdom

Americas

  +	Canada

  +	United States

Eastern Mediterranean

  + Afghanistan

Western Pacific

  + Korea

  + Japan

South-East Asia

  + None currently available

Africa

  + None currently available

We are working to include more countries.

> ## Development

Reviewer: maybe add sentence that help is appreciated, see issues? or describe some domains where help is appreciated (new data sources, geo-coding, ....)?

## Installation

Install the package and all dependencies with:

```r
remotes::install_github("epiforecasts/NCoVUtils", dependencies = TRUE)
```

## Usage

### Worldwide data

There are three sources of worldwide, country-level data on cases. One also includes deaths.

> 1. Extract total global cases and deaths by country using:

Reviewer: what is the source of the data for the `get_total_cases` function?

  + ```NCoVUtils::get_total_cases()```

> 2. Extract international case counts compiled by the WHO using:

Reviewer: ```suggestion
2. Extract daily international case and death counts compiled by the WHO using:
```

  + ```NCoVUtils::get_who_cases()```

> 3. Extract international case counts compiled by ECDC using:

Reviewer: ```suggestion
3. Extract daily international case and death counts compiled by ECDC using:
```

  + ```NCoVUtils::get_ecdc_cases()```

A further function for worldwide data extracts non-pharmaceutical interventions by country:

* ```NCoVUtils::get_interventions_data()```

And anonymised international patient linelist data can be imported and cleaned with:

* ```NCoVUtils::get_linelist()```

### Sub-national data

> We currently have functions to extract sub-national level data for 11 countries. These are typically at the admin-1 level, the largest regions available. We are also working on joining the data to standard georeferencing codes to allow easy mapping.

Reviewer: I wouldn't put the number of countries, we will forget to update it.

> We currently have functions to extract sub-national level data for 11 countries. These are typically at the admin-1 level, the largest regions available. We are also working on joining the data to standard georeferencing codes to allow easy mapping.

Reviewer: ```suggestion
We currently have functions to extract sub-national level data for 11 countries. These are at the most detailed subdivision level available, typically adm1 (provinces, regions, states or similar). We are also working on adding standard georeferencing codes to allow easy mapping.
```

> For most countries, subnational cases can be extracted by using the following:

Reviewer: I wouldn't say this, there is too many exceptions...
Maybe we should work on a standardized interface and/or a wrapper function that takes "country" as an argument and calls the respective function.

Reviewer: > Maybe we should work on a standardized interface and/or a wrapper function that takes "country" as an argument and calls the respective function.

Added an issue for this.

* ```NCoVUtils::get_[country]_regional_cases()```

But exceptions are given  below.

Currently we include functions for sub-national data in the following countries:

Europe

  +	Belgium

  +	France

  + Germany

  +	Italy

  +	Spain

  + United Kingdom - use `NCoVUtils::get_uk_nhs_region_cases()`

Americas

  +	Canada

  +	United States - use `NCoVUtils::get_us_regional_cases()`

Eastern Mediterranean

  + Afghanistan

Western Pacific

  + Korea

  + Japan

South-East Asia

  + None currently available

Africa

  + None currently available

> We are working to include more countries.

Reviewer: ```suggestion
We are working to include more countries.
Contributions via Pull Request are most welcome.
```
